templates/default/rt/rtqueuelist.html

```diff
@@ -110,17 +110,16 @@ <H1>{$layout.pagetitle}</H1>
 		</TD>
 		<TD class="lms-ui-buttons text-right nobr">
 			{if $queue.unread}
-			<a href="?m=rtqueueedit&id={$queue.id}&unread=0"><img src="img/ticket-read.png" alt="" title="{t a=$queue.unread}Mark as read ($a unread){/t}"></a>
+			{button type="link" href="?m=rtqueueedit&id={$queue.id}&unread=0" icon="lms-ui-icon-read" alt="Mark queue as unread"}
 			{/if}
 			{if $queue.deleted == 1 && $queue.deltime != 0 && (ConfigHelper::checkPrivilege('helpdesk_advanced_operations'))}
-			<A class="restore-rtqueue" HREF="?m=rtrestore&id={$queue.id}&qaction=restore"><IMG src="img/reload.gif" alt="{trans("Restore<!RT>")}" title="{trans("Restore<!RT>")}"></A>
-			<A class="remove-rtqueue" HREF="?m=rtremove&id={$queue.id}&qaction=delperm"><IMG src="img/delete.gif" alt="{trans("Delete permanently")}" title="{trans("Delete permanently")}"></A>
+			{button type="link" href="?m=rtrestore&id={$queue.id}&qaction=restore" icon="lms-ui-icon-reload" alt="Restore<!RT>"}
+			{button type="link" href="?m=rtremove&id={$queue.id}&qaction=delperm" icon="lms-ui-icon-delete" alt="Delete permanently" class="remove-rtqueue"}
 			{else}
```

Comment:
tutaj niepotrzebnie pełne nazwy klas.

Reply:
Usunąć całe class czy zostawić class="remove" ?

Reply:
Zostawiasz dla buttonów człon bez `lms-ui-icon-`.

Reply:
To już poprawiłem.